### PR TITLE
Add javadoc to getItemBarStep

### PR DESCRIPTION
**Pull request by Ueaj-Kerman**
The name is a bit unintuitive, but I can't think of anything better so I just added a javadoc

### CODE REVIEW

**Review by haykam821 — APPROVED** [review]

**Review by Mysterious-Dev — APPROVED** [review]

**Review by YanisBft — APPROVED** [review]

File: mappings/net/minecraft/item/ItemStack.mapping [hunk 1]
```diff
@@ -164,7 +164,9 @@ CLASS net/minecraft/class_1799 net/minecraft/item/ItemStack
 		ARG 1 otherStack
 	METHOD method_31578 isItemBarVisible ()Z
 	METHOD method_31579 getItemBarStep ()I
+		COMMENT The length of the 'full' section of the durability bar in pixels (out of 13).
```
Comment (liach) [inline mappings/net/minecraft/item/ItemStack.mapping]:
```suggestion
		COMMENT {@return the length of the filled section of the durability bar in pixels (out of 13)}
```

File: mappings/net/minecraft/item/ItemStack.mapping [hunk 2]
```diff
@@ -164,7 +164,9 @@ CLASS net/minecraft/class_1799 net/minecraft/item/ItemStack
...
 	METHOD method_31580 getItemBarColor ()I
+		COMMENT The color of the 'full' section of the durability bar.
```
Comment (liach) [inline mappings/net/minecraft/item/ItemStack.mapping]:
```suggestion
		COMMENT {@return the color of the filled section of the durability bar}
```

**Review by liach — APPROVED** [review]

**Comment by liach** [comment]
Imo "filled" might better convey that this is not totally full, but partially filled

**Review by enbrain — APPROVED** [review]

**Comment by Ueaj-Kerman** [comment]
merged the suggested changes

**Pull request merged**